packages/kbn-securitysolution-exception-list-components/src/header_menu/index.tsx

```diff
@@ -103,6 +103,7 @@ const HeaderMenuComponent: FC<HeaderMenuComponentProps> = ({
             </EuiButtonIcon>
           )
         }
+        onClick={(e) => e.stopPropagation()}
```

Comment:
`stopPopagation`=> as in List header the Accordion will open if the user clicked on the menu item